lib/entities/entry.ts

```diff
@@ -236,7 +236,7 @@ type EntryApi = {
   /**
    * Recursively collects references of an entry and their descendants
    */
-  references(maxDepth: number): Promise<EntryReferenceProps>
+  references({ maxDepth }: { maxDepth: number }): Promise<EntryReferenceProps>
```

Comment:
Probably okay for now, but if we add anymore query options we should export a `EntryReferenceOptions` interface

Reply:
Also does this arg need to be optional?

Reply:
In this case I'd add a new type ^